ggml/src/ggml-cuda/CMakeLists.txt

```
@@ -17,7 +17,7 @@ if (CUDAToolkit_FOUND)
         elseif(GGML_CUDA_F16 OR GGML_CUDA_DMMV_F16)
             set(CMAKE_CUDA_ARCHITECTURES "60;61;70;75;80")
         else()
-            set(CMAKE_CUDA_ARCHITECTURES "52;61;70;75;80")
+            set(CMAKE_CUDA_ARCHITECTURES "50;52;61;70;75;80")
```

```suggestion
            set(CMAKE_CUDA_ARCHITECTURES "50;61;70;75;80")
```

We are to my knowledge not using any features that were introduced with 5.1 or 5.2 so the code should be compiled only for 5.0.